lib/src/net/coap_exchange.dart

```diff
@@ -37,7 +37,7 @@ class CoapExchange {
   /// The current request
   CoapRequest? currentRequest;
 
-  /// The response
+  // The response
```

Comment:
Hmm, is it intentional that this is not a doc comment anymore?

Reply:
Not intentional, fixed: https://github.com/shamblett/coap/pull/76/commits/bf2e194af342174accb5b8c413397e7d3f129cdf